### DISCUSSION TITLE
Error: Process not in scope in Airbnb style guide on Xcode

### DISCUSSION THREAD

**Discussion by falcon-head** [airbnb/swift#230] [General]
# Issue with Airbnb Style Guide on Xcode - Cannot Find Type Process

## Description

I am facing an issue with the Airbnb Style Guide on Xcode. I have installed the package through Xcode UI by navigating to Files -> Add Package -> Github URL. However, when I try to build, it stops with an error "Cannot Find type Process". I am using Xcode 14.3 (14E222b) with Swift version 5.8.

![image](https://github.com/airbnb/swift/assets/32938743/31ef60f6-8e43-47aa-a401-6736938b52cc)
![image](https://github.com/airbnb/swift/assets/32938743/b28a83f6-4b50-4af0-8ac7-cc58138d2e50)
![image](https://github.com/airbnb/swift/assets/32938743/000a1e72-10b2-403f-b427-faa0d2e8766e)

## Solution Tried
I have tried the following solutions to resolve the issue:

- [x] Clearing cache and performing a clean build folder
- [x] Resetting package caches and resolving packages
- [x] Quitting Xcode and going out for a walk

Unfortunately, none of these solutions have resolved the issue.

I am seeking help from the community to resolve this issue. Has anyone else faced a similar issue, and if so, were you able to resolve it? Any help or suggestions would be greatly appreciated.

Thank you

**Comment by calda** [comment]
Can you share a sample Xcode project that demonstrates this issue? Thanks!

**Comment by falcon-head** [comment]
Hi @calda, thanks for your response. After trying to reproduce the issue on a new project, I found that the plugin works perfectly fine.
This makes me think that the issue might be related to my Xcode project rather than the plugin itself. 

I'll take a closer look at the build configuration and settings to see if there is anything that is causing the problem. If I still have any issues, I'll report this to Apple.

If I happen to find any workaround or solution, I'll update here. It might help others facing similar problems.

Thanks for your time!

**Comment by lawmaestro** [comment]
I ran into this very same issue. I've attached a sample project which exhibits this issue. It compiles fine for MacOS but not for iOS. As per [Apple's docs](https://github.com/airbnb/swift/discussions/**url**), Process is only available from Foundation on MacOS. So I'm unclear on how this build plugin can be made to work on an iOS project?
[TestFormatting.zip](https://github.com/airbnb/swift/files/12193275/TestFormatting.zip)

**Reply by lawmaestro** [reply]
To answer my own question here, the trick is to **not** checkbox your target when you add the package.
